tick/inference/hawkes_sumgaussians.py

```diff
@@ -120,6 +120,16 @@ class HawkesSumGaussians(LearnerHawkesNoParam):
     n_gaussians : `int`
         Number of gaussians used to approximate each kernel
 
+    max_mean_gaussian : `float`
```

Comment:
Not very important but it's always better if parameters documentations are in the same order as in the init function :)